fast/stages/01-resman/branch-teams.tf

```diff
@@ -94,6 +94,9 @@ module "branch-teams-team-dev-folder" {
     "roles/resourcemanager.projectCreator" = [
       module.branch-teams-dev-projectfactory-sa.iam_email
     ]
+    "roles/compute.xpnAdmin" = [
```

Comment:
Should we contemplate svpcs in teams folders? Unless I mistook what this does...

Reply:
this is just to allow attaching the project to the svpc. We'll replace it with custom role later (when everything works properly)